cmd/validate.go

```diff
@@ -36,10 +36,12 @@ var validateCmd = &cobra.Command{
 	Use:   "validate [env-name] [-f <file-or-dir>]",
 	Short: "Compare generated manifest against server OpenAPI spec",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if len(args) > 1 {
-			return fmt.Errorf("'validate' takes at most a single argument, that is the name of the environment")
+		if len(args) != 1 {
+			return fmt.Errorf("'validate' requires an environment name; use `env list` to see available environments")
 		}
+		env := args[0]
```

Comment:
There is a chance that args could be 0 since you only checking for != 1. This could panic.